src/containers/Tenant/Query/utils/replaceParams.ts

```diff
@@ -0,0 +1,3 @@
+export function replaceParams(template: string, params: Record<string, string>) {
+    return template.replace(/\${(\w+)}/g, (_, key) => params[key] || _);
+}
```

Comment:
Could you please add comment with example, what this function is for?

Reply:
added description and tests